fms_core/serializers.py

```diff
@@ -43,6 +53,41 @@ class Meta:
         fields = "__all__"
 
 
+class SampleExportSerializer(serializers.ModelSerializer):
+    individual_id = serializers.CharField(read_only=True, source="individual.label")
+    taxon = serializers.CharField(read_only=True, source="individual.taxon")
+    sex = serializers.CharField(read_only=True, source="individual.sex")
+    pedigree = serializers.CharField(read_only=True, source="individual.pedigree")
+    mother_id = serializers.SerializerMethodField()
+    father_id = serializers.SerializerMethodField()
+    container_kind = serializers.CharField(read_only=True, source="container.kind")
+    container_name = serializers.CharField(read_only=True, source="container.name")
+    container_barcode = serializers.CharField(read_only=True, source="container.barcode")
+    container_coordinates = serializers.CharField(read_only=True, source="container.coordinates")
+    location_barcode = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Sample
+        fields = ('biospecimen_type', 'name', 'alias', 'concentration', 'depleted', 'collection_site', 'tissue_source',
```

Comment:
When comparing to fields which are currently exported for samples by django admin and which are listed as needed by the analysis : 
- You seem to be missing the cohort field. 

- Also, the volume to be reported here should be derived from the volume_history field instead of the volume_used (last reported volume in the history). 

- The individual, mother and father id from the current export should return the (currently named) label field from individual instead of the proper id which is an internal identifier for which people have not much use.